src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/io/AprilTagPhotonCameraIO.java

```diff
@@ -48,6 +51,11 @@ protected void addSimCamera(Transform3d robotToCamera) {
         FieldConstants.VISION_SIMULATION.addCamera(cameraSim, robotToCamera);
     }
 
+    private PhotonPipelineResult getLatestPipelineResult() {
+        final List<PhotonPipelineResult> unreadResults = photonCamera.getAllUnreadResults();
+        return unreadResults.isEmpty() ? new PhotonPipelineResult() : unreadResults.get(unreadResults.size() - 1);
```

Comment:
i think it would be cleaner not to use a tenary operator here, but it's up to you